playgrounds/jquery/index.html

```diff
@@ -9,10 +9,26 @@
     <div id="examples">
         <script src="../../packages/jquery/lib/bundle.debug.js" charset="utf-8"></script>
 
+        <button id="cmp-btn">Switch Value</button>
         <div id="cmp">
             <script>
                 (function() {
-                    new window.DevExpress.ui.dxRadioButton(document.getElementById('cmp'), { defaultChecked: false, value: 'radio-example', label: 'Check me' })
+                    const instance = new window.DevExpress.ui.dxRadioButton(document.getElementById('cmp'), {
+                        checked: false,
+                        value: 'radio-example',
+                        label: 'Check me',
+                        onClick() { // not fired if checked is true
```

Comment:
Minor suggestion.

If we know why the ```onClick``` doesn't fire in this case, could you please provide more details in a ```NOTE``` comment?

Reply:
Unfortunately we know nothing useful at this point. Only condition - 'checked is true'